package.json

```diff
@@ -49,7 +49,6 @@
     "babel-preset-react": "6.3.13",
     "blankie": "1.1.1",
     "boom": "3.1.1",
-    "css-loader": "0.23.1",
```

Comment:
Gotta be careful with webpack loaders sometimes since it may just be used without you will be able to find where its being required